src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {
+  CSSTransition,
+  TransitionGroup,
+} from 'react-transition-group';
+import './TodoList.scss';
+import { TodoItem } from '../TodoItem';
+import { Todo } from '../../types/Todo';
+
+export const USER_ID = 6980;
+
+interface Props {
+  visibleTodos: Todo[];
+  deleteTodo: (idToDelete: number) => Promise<unknown>;
+  creating: boolean;
+  processings: number[];
+  tempTodo: Todo | null;
+  onUpdate: (
+    idToUpdate: number,
+    newTitle: string
+  ) => void | Promise<void | Partial<Todo>>,
+}
+
+export const TodoList: React.FC<Props> = ({
+  visibleTodos,
+  deleteTodo,
+  creating,
+  tempTodo,
+  processings,
+  onUpdate,
+}) => {
+  const handleUpdate = (idToUpdate: number | 0) => {
+    return (newTitle: string) => onUpdate(idToUpdate, newTitle);
+  };
+
+  // const onDelete = (idToDelete) => {
```

Comment:
Please remove the commented code if it's not necessary. If you plan to use it later, add a `// TODO:` comment with an explanation. 🗑️